com.kms.cura.dal/src/com/kms/cura/dal/database/UserDatabaseHelper.java

```diff
@@ -1,28 +1,66 @@
 package com.kms.cura.dal.database;
 
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
 import java.sql.Date;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.text.SimpleDateFormat;
 import java.util.Map;
 import java.util.Map.Entry;
 
+import javax.imageio.ImageIO;
+
+import org.apache.commons.codec.binary.Base64;
+
 import com.kms.cura.dal.exception.DALException;
+import com.kms.cura.dal.exception.DuplicatedUserEmailException;
 import com.kms.cura.dal.mapping.UserColumn;
+import com.kms.cura.dal.user.DoctorUserDAL;
+import com.kms.cura.dal.user.PatientUserDAL;
+import com.kms.cura.entity.ImageEntity;
+import com.kms.cura.entity.user.DoctorUserEntity;
+import com.kms.cura.entity.user.PatientUserEntity;
 import com.kms.cura.entity.user.UserEntity;
 
 public class UserDatabaseHelper extends DatabaseHelper {
+	private static String PATH = "D:\\cura-server\\image\\";
```

Comment:
Don't use absolute path for file like this. Instead use a system properties of Java described in http://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html